_overviews/toolkit/web-server-websockets.md

```diff
@@ -0,0 +1,118 @@
+---
+title: How to use websockets?
+type: section
+description: Using websockets with Cask
+num: 35
+previous-page: web-server-input
+next-page: web-server-cookies-and-decorators
+---
+
+{% include markdown.html path="_markdown/install-cask.md" %}
+
+You can create a WebSocket endpoint by using the `@cask.websocket` annotation. The endpoint method can return either a
+`cask.WsHandler` instance defining how the communication should take place, or a `cask.Response`, which rejects the 
+attempt at forming a WebSocket connection.
```

Comment:
```suggestion
You can create a WebSocket endpoint with the `@cask.websocket` annotation. The endpoint method should return a
`cask.WsHandler` instance defining how the communication should take place. It can also return a `cask.Response`, which rejects the 
attempt at forming a WebSocket connection.
```